runtime/rasdump/javadump.cpp

```diff
@@ -497,7 +497,9 @@ JavaCoreDumpWriter::JavaCoreDumpWriter(
 	bufferWrites = !_AvoidLocks
 			&& J9_ARE_NO_BITS_SET(_Context->eventFlags, J9RAS_DUMP_ON_GP_FAULT | J9RAS_DUMP_ON_ABORT_SIGNAL)
 			&& J9_ARE_ALL_BITS_SET(_Agent->prepState, J9RAS_DUMP_GOT_EXCLUSIVE_VM_ACCESS);
-
+#if (JAVA_SPEC_VERSION >= 24) && defined(open)
+#undef open
+#endif /* (JAVA_SPEC_VERSION >= 24) && defined(open) */
```

Comment:
This should not be version-sensitive.
This deserves a comment explaining why this is necessary.
A blank line before (i.e. keep line 500) and after would be welcome.

Have you considered adding this `#undef` to `ContinuationHelpers.hpp` instead? Today it only seems to impact this source file, but there's no guarantee the effects will be always be so limited. If `ContinuationHelpers.hpp` were included before `TextFileStream.hpp` this change would not work.

Reply:
updated PR to move undef to `ContinuationHelpers.hpp`, also added comment to explain the reason for such change.